### ISSUE TITLE
tracking info in tool for parser

### ISSUE THREAD

**Issue by vineetabajaj** [MAECProject/MAECProject.github.io#90]
I am trying to track how a tool run, and maintain its log and giving output in xml.
where i can find these parser without using cuckoo, please help me in tracking info so that i can make use of parser for different tool.

**Comment by ikiril01** [comment]
Hi @vineetabajaj - can you please provide some clarification about what you're trying to do? Is your goal to output MAEC from other tools (not just Cuckoo), or something else?

**Comment by vineetabajaj** [comment]
yes this is what i am trying to know how it works ..

**Comment by vineetabajaj** [comment]
yes i want to know how a tool store its output and maintain its log and gives output in xml(maec).

**Comment by vineetabajaj** [comment]
yes i want to know how a tool used by cuckoo store its output and maintain its log and gives output in xml(maec).

**Comment by ikiril01** [comment]
The Cuckoo Sandbox MAEC module [1] parses the Cuckoo results and then uses the `python-maec` library [2] to generate its output. 

[1] https://github.com/cuckoosandbox/cuckoo/blob/5231ff3a455e9c1c36239a025a1f6840029a9ed8/modules/reporting/maec40.py
[2] https://github.com/MAECProject/python-maec
